### DISCUSSION TITLE
Speech

### DISCUSSION THREAD

**Discussion by leocavalcante** [opencodeco/.github#3] [Announcements]
Eu não faço nada sozinho, vocês também não e **as organizações muito menos**; elas dependem de nós, estão em ombros de gigantes: indivíduos como nós que juntos colaboram. Nada mais justo e enriquecedor do que propiciar um ambiente livre para colaboração. Um lugar sem barreiras, sem fricção, sem impedimentos para as pessoas se sentirem à vontade para colaborar com os mais diversos projetos de código-aberto que usamos e temos acesso no nosso dia-a-dia. Vários desses que as próprias empresas usam, lucram e não retornam 1 centavo.

**Comment by joubertredrat** [comment]
Exato, eu gosto muito da minha visão de que "engenharia de software não é somente sobre código, é sobre pessoas e colaboração" que justamente remete aos meios colaborativos que usamos para discutir, obter conhecimento e compartilhar com todos

**Comment by flavioheleno** [comment]
muito bom, curti :-D
